R/downloadPR.R

```diff
@@ -25,15 +28,32 @@
 #' pr <- getPR('Belize')
 #' pr
 #' dir = tempdir()
-#' downloadPR(pr, year=2000, dir=dir)
+#' downloadPR(pr, year=2000, database = "FCC", dir=dir)
 #' 
 #' }
 #' 
-#' 
+#' ftp://ftp.glcf.umd.edu/glcf/LandsatTreecover/WRS2/
 #' @export downloadPR
 
-downloadPR <- function(pr, year, dir, log=NULL, baseURL='ftp://ftp.glcf.umd.edu/glcf/LandsatTreecover/WRS2/') {
-  
+dir <- 'data/'
+pr <- getPR('Belize')
+year <- 1990
+database = "FCC"
+
+downloadPR <- function(pr, year, database, dir, log=NULL, baseURL = baseURL) {
```

Comment:
Can you reduce the base URL to ftp://ftp.glcf.umd.edu/glcf/ and infer the rest of it from the year. Also I am wondering whether the database argument is needed since it can be inferred from the year. right?